src/main/java/core/basesyntax/userinvalidexeption/UserInvalidExeption.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax.userinvalidexeption;
```

Comment:
there can be more other exceptions in the future, so let's rename it to put them there instead of creating a separate package for each exception
```suggestion
package core.basesyntax.exception;
```